pytest_asdf/plugin.py

```diff
@@ -25,6 +25,12 @@ def pytest_addoption(parser):
         type="bool",
         default=False,
     )
+    parser.addini(
+        "asdf_schema_validate_default",
+        "Set to true to enable validation of the schema 'default' property",
+        type="bool",
+        default=True,
```

Comment:
Shouldn't the default here be `False`, as that was the previous behavior?

Reply:
Or do we not care since this does not affect runtime code and is only an issue for testing?

Reply:
The previous behavior wasn't quite `False` because it was successfully checking non-object defaults against the schema.  I'm not sure there's a good answer.